kamailio/local.cfg

```diff
@@ -41,6 +41,18 @@
 ##     in the zone that this server will service.
 #!substdef "!MY_AMQP_URL!kazoo://guest:guest@127.0.0.1:5672!g"
 
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL
+##     This should be the failover RabbitMQ server 
+##     in the zone that this server will service.
+##     Remove all but the last '#' to enable.
+# # #!substdef "!MY_AMQP_SECONDARY_URL!kazoo://guest:guest@127.0.0.1:5672!g"
+
+## CHANGE "kazoo://guest:guest@127.0.0.1:5672" TO THE AMQP URL and uncomment
+##     This should be the secondary failover RabbitMQ server 
+##     in the zone that this server will service.
```

Comment:
In that case, can you help me understand what this is, and how to document it?
/etc/kazoo/kamailio/default.cfg

## amqp connections
modparam("kazoo", "amqp_connection", "MY_AMQP_URL")
#!ifdef MY_AMQP_SECONDARY_URL
modparam("kazoo", "amqp_connection", "MY_AMQP_SECONDARY_URL")
#!endif
#!ifdef MY_AMQP_TERTIARY_URL
modparam("kazoo", "amqp_connection", "MY_AMQP_TERTIARY_URL")
#!endif